### PR TITLE
Release 2023.01.0

### PR DESCRIPTION
**Pull request by nmerket**
# Release Checklist

* [x] Make a `release_vX.Y` branch for your release changes.
* [x] Update `__version__.py` to reflect the new release version.
* [x] Create changelog for the new version. 
  * [x] Copy `docs/changelog/changelog_dev.rst` to `docs/changelog/changelog_X_Y.rst`.
  * [x] Remove the example change, update version names, and release date in `changelog_X_Y.rst`
  * [x] Clear out the dev changelog for the next version.
  * [x] Add a link to TOC in `docs/changelog/index.rst`
* [x] Write a migration guide for the new version. Link from TOC.
* [x] Build docs locally and ensure they render correctly.
* [x] Verify integration tests are working on CI
* [x] On Eagle
  * [x] Download the new default singularity image.
  * [x] Build a new environment
  * [x] Run a test batch
* [x] Tag release, make release in GitHub
* [x] Update RTD to include latest release
* [x] Build environment for release on Eagle

### CODE REVIEW

File: docs/changelog/migration_2023_01_0.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+=======================================
+What's new in buildstockbatch 2023.01.0
+=======================================
+
+.. admonition:: About this Document
+
+    This document describes changes between buildstockbatch version 2022.10.1 and
```
Comment (joseph-robertson) [inline docs/changelog/migration_2023_01_0.rst]: 2022.12.0

File: docs/changelog/migration_2023_01_0.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+See :doc:`changelog_2023_01_0` for details of changes since the last version.
+
+Residential HPXML Workflow Changes
+==================================
+
+Two new optional arguments were added to the Residential HPXML Workflow
+Generator. It is recommended to add the ``timeseries_num_decimal_places``
+argument and set it to at least ``5`` to reduce cumulative rounding errors in
+the timeseries outputs.
```
Comment (joseph-robertson) [inline docs/changelog/migration_2023_01_0.rst]: The second new optional argument is `include_timeseries_unmet_hours`.

File: docs/changelog/migration_2023_01_0.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+buildstockbatch. The default version of OpenStudio was changed to 3.5.1, so if
+you require an older version, make sure it is called out in your project file.
+
+See :doc:`changelog_2023_01_0` for details of changes since the last version.
+
+Residential HPXML Workflow Changes
+==================================
+
+Two new optional arguments were added to the Residential HPXML Workflow
+Generator. It is recommended to add the ``timeseries_num_decimal_places``
```
Comment (joseph-robertson) [inline docs/changelog/migration_2023_01_0.rst]: @nmerket Are you basing this recommendation on @rajeee's comment [here](https://github.com/NREL/resstock/issues/1016#issuecomment-1376693846)? Perhaps we want to change this recommendation to 3 based on his more recent comment [here](https://github.com/NREL/resstock/issues/1016#issuecomment-1404212345)?

Comment (nmerket) [reply docs/changelog/migration_2023_01_0.rst]: Yeah, that's what I was thinking. I'm going to sidestep this by referring people to the workflow generator documentation instead.

**Pull request merged**